specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json

```diff
@@ -0,0 +1,2549 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EventSourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "eventSourceUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EventSourceUpdateParameters"
+            },
+            "description": "Request object that contains the updated information for the event source."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The event source definition was successfully updated and is in the response.",
+            "schema": {
+              "$ref": "#/definitions/EventSourceResource"
+            }
+          },
+          "default": {
+            "description": "HTTP 404 (Not Found): The subscription, resource group, environment, or event source could not be found.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "EventSources"
+        ],
+        "operationId": "EventSources_Delete",
+        "x-ms-examples": {
+          "DeleteEventSource": {
+            "$ref": "./examples/EventSourcesDelete.json"
+          }
+        },
+        "description": "Deletes the event source with the specified name in the specified subscription, resource group, and environment",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EventSourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The event source was successfully deleted."
+          },
+          "204": {
+            "description": "The event source was successfully deleted."
+          },
+          "default": {
+            "description": "HTTP 404 (Not Found): The subscription, resource group, environment, or event source could not be found.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.TimeSeriesInsights/environments/{environmentName}/eventSources": {
```

Comment:
>eventSources [](start = 142, length = 12)

Are you going to need paging on eventSources?

Reply:
I don't think people will have a number of Event Sources on a single Environment that would activate paging.

Would it hurt if we leave the x-ms-pageable for Event Sources as well? I'm thinking of standardizing.